src/Microsoft.Azure.WebJobs.Host/Loggers/Logger/Constants/LogConstants.cs

```diff
@@ -123,7 +123,12 @@ public static class LogConstants
         /// Gets the name of the key used to store the EventId of the log message.
         /// </summary>
         public const string EventIdKey = "EventId";
-
+        
+        /// <summary>
+        /// Gets the name of the event name of the  log message.
+        /// </summary>
+        public const string EventName = "EventName";
```

Comment:
Can you change this to `EventNameKey` to match the others?

Reply:
I think i can abandon this PR, as I @cgillum has already raised a simialr PR, @cgillum  can you please take care of these comments in your PR https://github.com/Azure/azure-webjobs-sdk/pull/2544/files